### PR TITLE
Adds folder structure

### PR DESCRIPTION
**Pull request by doesitscript**
Addresses issue #1. 

Suggesting to use for presentation content format:

```
"YYYY-Presentations" 
```

Suggesting to use for subfolder content:

```
"MM - Topic (Presenter)"  
```
- Makes modification to suggested format for presentation content. 
- Allows for a single pane view of the years presentations.

_Note: If merging, please refer to issue that this closes. e.g. Closes #<number>_

_Not merging own work to allow for review._

### CODE REVIEW

**Comment by michaeltlombardi** [comment]
Couple questions and notes:

## Questions
1. Why are there gitignore files in every subfolder?
2. Is it preferable to have a subfolder in /Events/Meetups/2016 for each month and then the presentations of each month in that subfolder? As is, you end up with a third subfolder for March to hold the takeaways document or a takeaways document for every month in `/Events/Meetups/2016-Presentations`.
3. It looks like speaker bios ended up in a subfolder under presentations - is this intentional? If so, why?
4. This is nitpicky, but is it better to have the folder names capitalized or left all lowercase? It probably doesn't matter for people using windows machines but *nix boxes are case sensitive.

## Notes
- The `Automation-Manamement-Summit` folder needs to be renamed to `Automation-Management-Summit`
- I've updated #1 with an updated folder structure based on your inputs here.
- I have a strong preference for avoiding spaces in file and folder names wherever possible - it prevents mistyping and strange path problems on *nix systems and occasionally Windows ones as well. It also keeps URLs consistent.
- I have no real preference for naming month folders by name or by number. Whichever seems more clear should win.

Thanks for submitting the PR and doing the up-front work! I think if we can resolve the problems and address the concerns above this PR could be merged to master.

**Comment by doesitscript** [comment]
Requested changes made with exception of changing away from CamelCase. 

### Justification

Too tired. <Sarcasm>Feel free to fork my fork, make a pull request to my fork, I will then review and merge changes... you get the picture </Sarcasm>

File: Events/Meetups/2016/April/agenda.md [hunk 1]
```diff
@@ -0,0 +1 @@
+#agenda info
```
Comment (doesitscript) [inline Events/Meetups/2016/April/agenda.md]: ignore this.

**Comment by doesitscript** [comment]
Merged in your changes. Also I added the New PSUG icon.

**Comment by michaeltlombardi** [comment]
Pinged the team on #14 since you're also addressing that in this PR.

**Comment by michaeltlombardi** [comment]
Good job! We're accepting this PR now. :) Congrats on submitting the first one.

**Pull request merged**